oneview_redfish_toolkit/tests/blueprints/test_manager_collection.py

```diff
@@ -43,19 +44,23 @@ def setUpClass(self):
         self.app.register_blueprint(
             manager_collection.manager_collection)
 
+    @mock.patch.object(multiple_oneview, 'get_map_appliances')
     def test_get_manager_collection_unexpected_error(
-            self):
+            self, get_map_appliances):
         """Tests ManagerCollection with an error"""
 
-        self.oneview_client.appliance_node_information.get_version.side_effect = \
-            Exception()
+        get_map_appliances.side_effect = \
+            Exception("An exception was occurred")
 
         with open(
                 'oneview_redfish_toolkit/mockups/errors/'
                 'Error500.json'
         ) as f:
             error_500 = json.load(f)
 
+        error_500_excep = copy.deepcopy(error_500)
+        error_500_excep["error"]["message"] = "An exception was occurred"
```

Comment:
My english is not so good, but the message should be *"An exception ~was~ has occurred"*, not?

Reply:
Yes, that is correct! I will fix it. Thanks 😄